### DISCUSSION TITLE
how-to-automatically-backup-wordpress-posts-to-github/

### DISCUSSION THREAD

**Discussion by giscus** [JakeSteam/blog-programming#24] [Comments]
# how-to-automatically-backup-wordpress-posts-to-github/

When programming, most developers use GitHub (or another hosted git solution) to make sure all of their work is backed up in multiple places to ensure it is never lost. If you’re anything like me, when writing blog posts you want the same peace of mind you have with the rest of your work. This post will cover how to automatically export all WordPress posts to GitHub (and keep it updated), so that they can be imported later if necessary.

https://blog.jakelee.co.uk/how-to-automatically-backup-wordpress-posts-to-github/

**Comment by DavDaddy** [comment]
I'm interested in this but noticed that the plugin hasn't been updated in a long time.  Can you confirm this still works and there's no (known) security issues? 

I just got done going 12 rounds with someone determined to hack my site so I'm a little gun shy at the moment.

Thx
